.gitignore

```diff
@@ -6,6 +6,8 @@ dbd/
 configure/*.local
 html/
 **/O.*
+O.*
 envPaths
 QtC-*
 *.orig
+*.log
```

Comment:
We tee the build output to make.log or build.log to make debugging build failures easier.

Cheers,

- Bruce


On 09/01/2016 12:02 AM, Ralph Lange wrote:
>
> In .gitignore <https://github.com/epics-base/pvaSrv/pull/52#discussion_r77123254>:
>
> >  envPaths
> >  QtC-*
> >  *.orig
> > +*.log
>
> What is creating *.log files?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/epics-base/pvaSrv/pull/52/files/83d2debe492789555a5c561654c83bd53bcdadfd#r77123254>, or mute the 
> thread <https://github.com/notifications/unsubscribe-auth/AF43JiJ3Fv8NztgD1rvwEE2Ocz5iOQacks5qlnhzgaJpZM4JyUF0>.
>

-- 
Bruce Hill
Member Technical Staff
SLAC National Accelerator Lab
2575 Sand Hill Road M/S 10
Menlo Park, CA  94025